common/persistence/nosql/nosqlplugin/cassandra/domain.go

```diff
@@ -410,6 +410,10 @@ func (db *cdb) SelectAllDomains(
 		replicationClusters = []map[string]interface{}{}
 		badBinariesData = []byte("")
 		badBinariesDataEncoding = ""
+		isolationGroups = []byte("")
+		isolationGroupsEncoding = ""
+		asyncWFConfigData = []byte("")
```

Comment:
`asyncWFConfigData = nil` would also do the job. not sure why the byte arrays were reset like this before.